Header header
kinect/NiteSkeleton[] skeletons
